packages/falso/src/lib/user.ts

```diff
@@ -46,7 +46,7 @@ export function randUser<Options extends FakeOptions>(options?: Options) {
       email: randEmail(),
       firstName: randFirstName(),
       lastName: randLastName(),
-      phone: randPhoneNumber(),
+      phone: <string>randPhoneNumber(),
```

Comment:
It should be string, why do you need the casting?

Reply:
It is fixed